libbeat/filter/config.go

```diff
@@ -0,0 +1,14 @@
+package filter
+
+type DropFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type IncludeFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type FilterConfig struct {
+	DropFields    *DropFieldsConfig    `config:"drop_fields"`
```

Comment:
While thinking how to restructure the config file, I came up with the idea to rename `include_fields` with `drop_all_fields_except`. The name is rather long, but it describes exactly what it's designed for. I can do this change in a separate PR.